README.md

```diff
@@ -1,2 +1,45 @@
-# HW01
-Edit the readme
+---
+output:
+  html_document: default
+  pdf_document: default
+---
+# My life in Quarantine
+### Althea Bock-Hughes
+
+## Quarantine projects included:
+
+* Knitting
+
+* Baking Sourdough Bread
+
+* Watching "Avatar the Last Airbender" for the first time 
```

Comment:
Nice job incorporating lists into your bio! I have never watched Avatar so we should watch it! I keep getting harrassed about it haha :)